src/main/resources/logback-spring.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+
+	<property name="PROV_HIST_HOME" value="opt/provision/history/logs" />

```

Comment:
Is `opt/provision/history/logs` missing a leading slash?

Reply:
Top.. Funny enough it Works..   but i fix it.